fe/fe-core/src/main/java/org/apache/doris/nereids/trees/expressions/functions/agg/ApproxTopK.java

```diff
@@ -55,9 +55,19 @@ public ApproxTopK(boolean distinct, boolean alwaysNullable, Expression... varArg
 
     @Override
     public void checkLegalityBeforeTypeCoercion() {
-        if (arity() < 1) {
+        if (arity() < 3) {
```

Comment:
Why change arguments num from 1 to 3?

Reply:
The existing function framework code cannot implement approx_top_k and approx_top_sum with default values.